andino_hardware/README.md

```diff
@@ -29,6 +29,13 @@ Some frequent errors:
  - If one of the motors rotates in the opposite direction (think about the orientation of the motors in the chassis) probably the output(+ and -) of the L298N's output should be toggled.
  - When moving forward the encoder values should increase while moving backwards they should decrease. If it is happening the other way around probably the A and B encoder signals should be toggled.
 
+### Raspberry-Power
+
+<img src="docs/andino_diagram_raspberry.jpg" />
+
+*NOTE: depending on how good is the powerbank we have, the motors should be powered with more or less voltage. While a higher voltage value (up to 9V) leads to smoother operation (better motor speed control), it also increases their open-loop speed, which is noticeable particularly during motion start and varies according to the power bank quality (output current capabilities). Therefore, adjusting the output voltage to lower values (around 7V) may be required so as to make the motors work as expected.
```

Comment:
What about `depending on the power bank maximum output current, the motors may need to be powered with a voltage value less than 9V. While a higher...`?